### ISSUE TITLE
No helpful information for user when data partition is unavailable on server

### ISSUE THREAD

**Issue by ebuchlin** [Helioviewer-Project/helioviewer.org#251]
Following a power outage at IAS, a partition containing the AIA JP2 files was not mounted. During that time, http://helioviewer.ias.u-psud.fr/ with an AIA layer just showed a black screen instead of the image, with no indication for the user of what could be wrong. There were items in the Image Timeline, but with "broken link" icons instead of the previews (see screenshot)

![screenshot](https://user-images.githubusercontent.com/3624676/47778929-3b518b00-dcf8-11e8-889f-6d5cc6d5a147.png)

This is not sufficient for the user to guess that something is broken and that he should contact us to solve the issue.

There should be some kind of explicit information for the user when data files are not available.

**Comment by wafels** [comment]
Not sure I understand this. The timeline is reporting data, but there are no images to create the thumbnails?

**Comment by ebuchlin** [comment]
Yes, the data files were not accessible, so there were no images or thumbnails. The timeline is based on the database, which was accessible and containing the correct metadata, so it was reporting data. All this is quite logical; the issue is that in such a case the user is not told why there are no images or thumbnails, while their presence is to be expected.

**Comment by dgarciabriseno** [comment]
We should update this so that when these API calls return errors errors, we display a message saying a problem on our side has occurred.

**Comment by dgarciabriseno** [comment]
We should still do this.

**Comment by dgarciabriseno** [comment]
This may have been fixed by #660 

If the server throws a 500 error, then an error message will show up. It would be good to verify if that covers this issues.
